File: DsWorkNet/TestWork/TestWork/Service/DemoService.cs
using System;

using Dswork.Core.Db;

using TestWork.Dao;
using TestWork.Model;

//using Spring.Objects.Factory.Attributes;
//using Spring.Stereotype;

namespace TestWork.Service
{
	//[Service]
	public class DemoService : BaseService<Demo, long>
	{
		//[Autowired]
		// 这里的virtual和protected get非常重要，否则无法直接使用demoDao
		public virtual DemoDao demoDao { set; protected get; }
		//protected virtual DemoDao GetDemoDao() { return demoDao; }

		protected override EntityDao<Demo, long> GetEntityDao()
		{
			return demoDao;
		}
		/*
		public override int Save(Demo entity)
		{
			base.Save(entity);
			base.Save(entity);
			return 1;
		}
		*/
		public Demo GetShow(long id)
		{
			return demoDao.GetShow(id);
		}
	}
}

File: AndroidWeb/src/testwork/service/demo/DemoService.java
/**
 * DEMOService
 */
package testwork.service.demo;

import java.util.Map;

import org.apache.commons.beanutils.ConvertUtils;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import dswork.core.db.EntityDao;
import dswork.core.db.BaseService;
import testwork.model.demo.Demo;
import testwork.dao.demo.DemoDao;

@Service
@SuppressWarnings("unchecked")
public class DemoService extends BaseService<Demo, Long>
{
	@Autowired
	private DemoDao dao;

	@Override
	protected EntityDao getEntityDao()
	{
		return dao;
	}
	
	public void updBatch(Map m)
	{
		String[] ids_s = m.get("ids").toString().split(",");
		Long[] ids = (Long[])ConvertUtils.convert(ids_s, Long.class);
		for(Long id: ids)
		{
			Demo po = new Demo();
			po.setId(id);
			po.setTitle(m.get("title").toString());
			po.setContent(m.get("content").toString());
			po.setFoundtime(m.get("foundtime").toString());
			dao.update(po);
		}
	}
	
	
}
